identity-iota/Cargo.toml

```diff
@@ -50,7 +50,10 @@ features = ["blake2b"]
 tokio = { version = "1.15", default-features = false, features = ["macros"] }
 
 [features]
-default = ["async"]
+default = ["arc", "async"]
+
+# Changes smart pointers in Resolver from Rc to Arc.
```

Comment:
Not sure if this should be done with feature flags. See [my comment to `get_client`](https://github.com/iotaledger/identity.rs/pull/594/files#r812729822).